code/modules/mob/living/living.dm

```diff
@@ -1911,7 +1911,7 @@
 	if(STAPER > 5)
 		ttime = 10 - (STAPER - 5)
 		if(ttime < 0)
-			ttime = 0
+			ttime = 1
```

Comment:
I'm going with my gamer senses and making a call that having the instant teleportation isn't as good as having a fast sweep. I think it being instant is more disorienting overall.